charts/k8ssandra/README.md

```diff
@@ -42,14 +42,12 @@ Provisions and configures an instance of the entire K8ssandra stack. This includ
 | cassandra.cassandraLibDirVolume.storageClass | string | `"standard"` | Storage class for persistent volume claims (PVCs) used by the underlying cassandra pods. Depending on your Kubernetes distribution this may be named "standard", "hostpath", or "localpath". Run `kubectl get storageclass` to identify what is available in your environment. |
 | cassandra.cassandraLibDirVolume.size | string | `"5Gi"` | Size of the provisioned persistent volume per node. It is recommended to keep the total amount of data per node to approximately 1 TB. With room for compactions this value should max out at ~2 TB. This recommendation is highly dependent on data model and compaction strategies in use. Consider testing with your data model to find an optimal value for your usecase.  |
 | cassandra.allowMultipleNodesPerWorker | bool | `false` | Permits running multiple Cassandra pods per Kubernetes worker. If enabled resources.limits and resources.requests **must** be defined. |
-| cassandra.heap | object | `{"newGenSize":null,"size":null}` | Cluster-level heap configuration |
-| cassandra.resources | object | `{}` | Resource requests for each Cassandra pod. |
+| cassandra.resources | object | `{}` | Cluster-level heap configuration size: newGenSize: -- Resource requests for each Cassandra pod. |
```

Comment:
Note that leaving the doc-style comment on a commented-out property causes the doc to be concatenated with the docs for the next property. Now it looks like `cassandra.resources` is for heap settings. If those properties are going to be commented out, then their doc comments must _not_ start with `--`.

Reply:
Here's how it looks in the generated doc:
![image](https://user-images.githubusercontent.com/8182938/107571246-a9b07380-6bb8-11eb-9cf4-a8c5a0baa5f6.png)

Reply:
Nice catch

Reply:
See latest comment to clarify handling in next PR update.